<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="10592" started-at="2016-07-20T15:48:59Z" finished-at="2016-07-20T15:49:09Z">
    <groups>
      <group name="domain">
        <method signature="VerifyWebComLogin.VerifyValidLogin()[pri:0, instance:br.rubens.teste.VerifyWebComLogin@35eaab]" name="VerifyValidLogin" class="br.rubens.teste.VerifyWebComLogin"/>
      </group> <!-- domain -->
    </groups>
    <test name="Default test" duration-ms="10592" started-at="2016-07-20T15:48:59Z" finished-at="2016-07-20T15:49:09Z">
      <class name="br.rubens.teste.VerifyWebComLogin">
        <test-method status="FAIL" signature="VerifyValidLogin()[pri:0, instance:br.rubens.teste.VerifyWebComLogin@35eaab]" name="VerifyValidLogin" duration-ms="10578" started-at="2016-07-20T12:48:59Z" description="Realiza o login na aplicação" finished-at="2016-07-20T12:49:09Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.fail(Assert.java:101)
	at br.com.rubens.framework.Util.waitForElement(Util.java:108)
	at br.com.rubens.page.LoginPageTest.LogarPageLogin(LoginPageTest.java:63)
	at br.rubens.teste.VerifyWebComLogin.VerifyValidLogin(VerifyWebComLogin.java:36)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyValidLogin -->
      </class> <!-- br.rubens.teste.VerifyWebComLogin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
